web/packages/teleterm/src/services/logger/loggerService.ts

```diff
@@ -220,3 +245,12 @@ function getRegularConsoleTransport(opts: FileLoggerOptions) {
     }),
   });
 }
+
+function getLoggerName(
```

Comment:
Did you mean to reuse this function in `getRegularConsoleTransport` as well?

Reply:
Oops, fixed.